python/custreamz/custreamz/kafka.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+import confluent_kafka as ck
+from cudf_kafka._lib.kafka import KafkaDatasource
+
+import cudf
+
+from custreamz.utils import docutils
+
+
+# Base class for anything class that needs to interact with Apache Kafka
+class CudfKafkaClient(object):
+    def __init__(self, kafka_configs):
+        self.kafka_configs = kafka_configs
+
+        self.kafka_confs = {}
+        for key, value in self.kafka_configs.items():
+            self.kafka_confs[str.encode(key)] = str.encode(value)
+
+        self.kafka_meta_client = KafkaDatasource(self.kafka_confs)
+
+    @docutils.doc_unsubscribe()
```

Comment:
Any reason you're putting these in a separate file? Would be better to just inline them here unless there's a reason not to.

Reply:
no particular reason. I will move them inline